packages/core/src/devtools/isDevToolsEnabled.ts

```diff
@@ -1,3 +1,9 @@
-export const isDevToolsEnabled = Boolean(
-  typeof window !== 'undefined' && window?.sessionStorage?.getItem('__GRIFFEL_DEVTOOLS__'),
-);
+export const isDevToolsEnabled: Boolean = (() => {
+  // Accessing "window.sessionStorage" causes an exception when third party cookies are blocked
+  // https://stackoverflow.com/questions/30481516/iframe-in-chrome-error-failed-to-read-localstorage-from-window-access-deni
+  try {
+    return Boolean(typeof window !== 'undefined' && window?.sessionStorage?.getItem('__GRIFFEL_DEVTOOLS__'));
```

Comment:
```suggestion
    return Boolean(typeof window !== 'undefined' && window.sessionStorage?.getItem('__GRIFFEL_DEVTOOLS__'));
```

You already have an undefined guard for `window`